packages/lib/modules/vebal/vote/Votes/MyVotes/MyVotesTable/VoteWeightInput.tsx

```diff
@@ -42,33 +49,40 @@ export function VoteWeightInput({
   const [isEditing, { toggle }] = useBoolean(false)
   const [editingValue, setEditingValue] = useState(parseFloat(percentage).toFixed(2))
 
+  const endOfLocking = dateTimeLabelFor(votingTimeLockedEndDate(lastVoteTime || 0))
+
   return (
     <VStack align="start" w="full">
-      <InputGroup>
-        <Input
-          autoComplete="off"
-          autoCorrect="off"
-          bg="background.level1"
-          max={max}
-          min={0}
-          onBlur={() => {
-            setEditingValue(parseFloat(percentage).toFixed(2))
-            toggle()
-          }}
-          onChange={handleChange}
-          onFocus={() => {
-            setEditingValue(parseFloat(percentage).toFixed(2))
-            toggle()
-          }}
-          onKeyDown={blockInvalidNumberInput}
-          type="number"
-          value={isEditing ? editingValue : parseFloat(percentage).toFixed(2)}
-          {...inputPros}
-        />
-        <InputRightElement pointerEvents="none">
-          <Percent color="grayText" size="20px" />
-        </InputRightElement>
-      </InputGroup>
+      <Tooltip
+        isDisabled={isTimeLocked}
+        label={`Your vote is timelocked. Once you vote on a pool, your vote is fixed for 10 days. No edits can be made until ${endOfLocking}`}
+      >
+        <InputGroup>
+          <Input
```

Comment:
maybe add `isReadOnly` so you really can not edit the field?

Reply:
The `inputProps` already contains and `isDisabled` attribute for that

Reply:
ah the check on ln 57 should be the other way round then i think

`isDisabled={!isTimeLocked}`

Reply:
you are right, fixed